script/Arcueid/Day 13/13_00_ARC13_6.txt

```diff
@@ -42,6 +42,7 @@ From deep within, I can hear something clattering around as it moves.
 [sha:65dc0e1a9a8be4635c000f7791f5784f0a0228ed]{
 -- Page 449, Offset 32970.
 -- ほんとうに、生き汚い。
+// group pr check: Disgusting -> holding on for dear life (in the most disgusting way), a discussion was had about a potential English equivalent. Pathetic maybe? we're unsure.
```

Comment:
maybe
> How pathetically you cling to life

Reply:
Feels a bit direct of an address imo. I'd originally considered something like 
> What cockroachlike tenacity

but it might be a bit of a jump. Maybe just your suggestion but more internally musing?
> It's honestly pathetic how desperately he clings to life

Reply:
I don't really feel either of those is particularly necessary given how simple the line is

Reply:
Main thing that rubs me the wrong way is the `you` I think.
> How pathetically he clings to life.

or
> It's pathetic the way he clings to life.

Reply:
> How pathetically he clings to life.

works for me